docs/managed-victoriametrics/quickstart.md

```diff
@@ -11,179 +11,265 @@ aliases:
 while the hardware spins-up, just wait for a couple of minutes and reload the page. 
-You'll also be notified via email once provisioning is finished:
+You'll also be notified via email once your deployment is ready to use:
 
 <p>
-  <img src="quickstart_deployment-created.png" width="800">
+  <img src="create_deployment_created.png" width="800">
 </p>
 
 <p>
-  <img src="quickstart_deployments-running.png" width="800">
+  <img src="create_deployment_active_email.png" width="800">
 </p>
 
-## Deployment access
+## Start writing and reading data
+
+After transition from `PROVISIONING` to `RUNNING` state, Managed VictoriaMetrics deployment is fully operational 
+and is ready to accept write and read requests. 
 
-After transition from `PROVISIONING` to `RUNNING` state, VictoriaMetrics is fully operational 
-and ready to accept write or read requests. But first, click on deployment name to get the access token:
+Click on deployment name and navigate to the Access tab to get the access token:
 
 <p>
-  <img src="quickstart_deployment-access-token.png" width="800">
+  <img src="deployment_access.png" width="800">
 </p>
 
 Access tokens are used in token-based authentication to allow an application to access the VictoriaMetrics API. 
 Supported token types are `Read-Only`, `Write-Only` and `Read-Write`. Click on token created by default 
 to see usage examples:
 
 <p>
-  <img src="quickstart_read-token.png" width="800">
+  <img src="deployment_access_write_example.png" width="800">
 </p>
 
 <p>
-  <img src="quickstart_write-token.png" width="800">
+  <img src="deployment_access_read_example.png" width="800">
 </p>
 
-Follow usage example in order to configure access to VictoriaMetrics for your Prometheus, 
+Follow usage examples in order to configure access to VictoriaMetrics for your Prometheus, 
 Grafana or any other software.
 
-## Modifying deployment
+## Modifying an existing deployment
 
-Remember, you always can add, remove or modify existing deployment by changing their size or any parameters on the 
-update form.
+Remember, you can always add, remove or modify existing deployment by changing its size or any parameters on the 
+deployment's page.
 
 <p>
-  <img src="quickstart_update-deployment.png" width="800">
+  <img src="modify_deployment.png" width="800">
 </p>
-
-There is another options present to customise you deployment setup. 
-To discover them click on `Customise` button
+ 
+To discover additional configuration options click on `Advanced Settings` button, so you should see the following:
 
 <p>
-  <img src="quickstart_customise-deployment.png" width="800">
+  <img src="modify_deployment_additional_settings.png" width="800">
 </p>
 
-In that section additional params can be set:
+In that section, additional params can be set:
 
 * `Deduplication` defines interval when deployment leaves a single raw sample with the biggest timestamp per each discrete interval;
```

Comment:
please add link to VM docs for deduplication

Reply:
link was added